### PR TITLE
Add regression bug tracking dashboard

### PR DESCRIPTION
**Pull request by alancutter**
Screenshot

This change required altering <cz-issues> to accept a query with both type and components which resulted in a mild refactoring. Please see the first commit for that change in isolation.

### CODE REVIEW

**Comment by shans** [comment]
That looks like a nice improvement in querying, thanks

File: client/cz-issues.html [hunk 1]
```diff
@@ -31,19 +31,21 @@
...
-            });
-        } else if (query.type == 'component') {
-          this.addCallbackToQuery(query, 'component', callback,
-            {
-              site: 'issues',
-              q: "component:" + query.component,
-            });
+        var queryParts = [];
+        if ('user' in query) {
+          queryParts.push('owner:' + query.user);
```
Comment (dstoc) [inline client/cz-issues.html:36]: `owner=`

File: client/cz-issues.html [hunk 2]
```diff
@@ -31,19 +31,21 @@
...
         }
+        if ('type' in query) {
+          queryParts.push('Type=' + query.type);
+        }
+        if ('components' in query) {
+          queryParts.push('component:' + query.components.join(','));
```
Comment (dstoc) [inline client/cz-issues.html:42]: Is this intentionally not `component=` ?

Comment (alancutter) [inline client/cz-issues.html:42]: Seems like both `=` and `:` are accepted. The auto complete on crbug.com prefers `:` for component and owner and `=` for type.

Comment (dstoc) [inline client/cz-issues.html:42]: > Seems like both = and : are accepted. The auto complete on crbug.com prefers : for component and owner and = for type.

They have different meanings. `=` means exact match, `:` means substring.

**Pull request merged**